feugy/tabulous
# feat(draughts): introduces draught public game!

### What's in there?

Draughts is a simple yet very challenging game. And it's very easy to implement!

Are included here:

- feat(draughts): introduces games!
- fix(web): can not return to home page from game

### How to test?

Start Tabulous, the game should be available in the public list.
Then you can create new game, and enjoy it!
